README.md

```diff
@@ -1,138 +1,125 @@
-        * default: http://Kong:8001
+# **Configuration**
 
-  * AUTH_TOKEN_EXP
-        * Expiration time in second for generated JWT tokens
-        * default: 420
+Before running the service, always check if the environment variables are suited to your
+environment.
 
-  * AUTH_TOKEN_CHECK_SIGN
-        * Whether Auth should verify received JWT signatures. Enabling this will cause one extra query to be performed.
-        * default: False
+| Name                  | Description                                                                                                   | Default value   | Accepted values
+| --------------------- | ------------------------------------------------------------------------------------------------------------- | --------------- | ---------------
+| AUTH_CACHE_DATABASE   | Cache database name (or number).                                                                              | 0               | string
+| AUTH_CACHE_HOST       | Address used to connect to the cache.                                                                         | redis           | hostname/IP
+| AUTH_CACHE_NAME       | Type of cache used. Currently only Redis is supported.                                                        | redis           | string
+| AUTH_CACHE_PWD        | Password to access the cache database.                                                                        | empty password  | string
+| AUTH_CACHE_TTL        | Cache entry time to live in seconds.                                                                          | 720             | number
+| AUTH_CACHE_USER       | Username to access the cache database.                                                                        | redis           | string
+| AUTH_DB_HOST          | Address used to connect to the database.                                                                      | http://postgres | hostname/IP
+| AUTH_DB_NAME          | Type of database used. Currently only PostgreSQL is supported.                                                | postgres        | string
+| AUTH_DB_PWD           | Database password.                                                                                            | empty password  | string
+| AUTH_DB_USER          | Database username.                                                                                            | auth            | string
+| AUTH_KONG_URL         | Kong location (If set to 'DISABLED' Auth won´t try to configure Kong and will generate secrets for the JWT tokens by itself). | http://kong:8001 | hostname/IP:port or DISABLE
+| AUTH_TOKEN_CHECK_SIGN | Whether Auth should verify received JWT signatures. Enabling this will cause one extra query to be performed. | False           | boolean
+| AUTH_TOKEN_EXP        | Expiration time in seconds for generated JWT tokens.                                                          | 420             | number
```

Comment:
There are others variables:
https://github.com/dojot/auth/blob/540919067d3e5da579b054aadaa219f2e189d2c9/auth/conf.py#L34
For example, the auth_email_*